test/units/testsuite-23.clean-unit.sh

```diff
@@ -114,6 +125,7 @@ test -L /var/log/test-service
 
 systemctl stop test-service
 
+
```

Comment:
Why this empty line inserted?

Reply:
Whoops :facepalm:

Reply:
still here

Reply:
*sigh*, I think GH mocks me today. I'm pretty sure I finally fixed it (attempt \#3), but now I can't push it:

```
$ git push -f origin test-merge 
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```